### ISSUE TITLE
[EN][DUNGEON] Scolomance

### ISSUE THREAD

**Issue by Helbringer** [Atlantiss/BugTracker#3721]
1. There is no invisible wall in (Kirtonos the Herald)s room, i have seen many groups fall down the cliff edge because of this. http://www.wowhead.com/npc=10506/kirtonos-the-herald
http://cata.openwow.com/npc=10506
![wowscrnshot_070215_143657](https://cloud.githubusercontent.com/assets/10423624/8479086/5ee29ac2-20cd-11e5-85eb-0d28fbf20fbf.jpg) When Kirtonos uses his knockback ability against a player near the fence, the player will be knocked out of the balcony completely, this should not happen. http://wow.gamepedia.com/Kirtonos_the_Herald

2. Jandice Barov does not use her clone ability properly. Her clones are still, dont attack and stand in lines, they do nothing for about 1 minuite and after that they desapwn and jandice is back to fighting normally. How this should work is that when she spawns her clones, they will all be aggresive and attack with the real jandice as well. On atlantiss the real jandice despawns as the clones spawn.
Evidence in this video https://youtu.be/PCbyX9KkaAM?t=96 taken in cataclysm. looking further into this issue i have found that whatever dev scripted her, must of accidently scripted her to post cataclysm version which was introduced in Mists of Pandaria. How its supposed to be in cata http://wow.gamepedia.com/Jandice_Barov_%28original%29 , this is how it is in Mists http://wow.gamepedia.com/Jandice_Barov . http://www.wowhead.com/npc=59184/jandice-barov .
![wowscrnshot_070215_144039](https://cloud.githubusercontent.com/assets/10423624/8479376/5779009e-20cf-11e5-9eef-41cf73055ff7.jpg) 
As you can see it resembles the Mists version.

3. NPC's have "simple" pathfinding in scholomance, like (hydrospawn) is scared of water, so are all the mobs in Scholomance. All NPC's in this instance are scared of water, they cannot move onto or move around it, this can cause some unusual problems such and flying through ceilings.
This happens on every room that has water present.
![wowscrnshot_070215_145309](https://cloud.githubusercontent.com/assets/10423624/8479483/fd76307a-20cf-11e5-95b5-c556a16346ff.jpg)
Basically, the mobs not being able to walk through water generally isnt a issue in Scholo as there isnt much water, however for the Npc's that stand on water, it can really make them very unblizzlike.
These NPC's affected by the pathfinding issue are, (Risen bonewarder), (Lady Illucia Barov), (Risen Guard), (Lord Alexei Barov). These mobs will always find their way upstairs, other mobs can also sometimes do this, on occations when faced with water.
![wowscrnshot_070215_145600](https://cloud.githubusercontent.com/assets/10423624/8479744/99c043c0-20d1-11e5-8c30-59d23d79440f.jpg)

4. The door to (Ras Frostwhisper) does not open at all, once the two bosses (Vectus) (Marduk Blackpool) have been killed, it does not open. http://www.wowhead.com/npc=10508/ras-frostwhisper
![wowscrnshot_070215_144618](https://cloud.githubusercontent.com/assets/10423624/8479892/89777c08-20d2-11e5-8083-ce67eb2c6e67.jpg)
![wowscrnshot_070215_144642](https://cloud.githubusercontent.com/assets/10423624/8479896/8e66e30c-20d2-11e5-957b-3d3bbdb3621e.jpg)
http://wowwiki.wikia.com/Ras_Frostwhisper
Because of this door not opening, it means that two of the quests in this instance are incompletable.
These quests are, http://wow.gamepedia.com/Quest:The_Lich,_Ras_Frostwhisper and http://wow.gamepedia.com/Quest:Barov_Family_Fortune . As Ras cant be killed and the final Deed cannot be collected.

5. (Darkmaster Gandling) no longer teleports players into rooms while fighting him, he seems to only cast "Arcane missiles" repeatedly. I stood next to him for about 5 minuites straight and noticed i did not get teleported. I first noticed this while in a RDF group and have been to the instance several times, never to see a player be teleported.
![wowscrnshot_070215_145950](https://cloud.githubusercontent.com/assets/10423624/8480265/aa61b5f8-20d4-11e5-9da0-1b070e7b7df7.jpg) http://wow.gamepedia.com/Darkmaster_Gandling_%28original%29

6. Finally i have also noticed that the loot tables for all of scholomance (mobs) not bosses are wrong, the reason for this is that the loot tables of the mobs are the same as from vanilla. As only level 55+ items seem to drop, like they would in vanilla when scholomance was for level 60's.
http://wowwiki.wikia.com/Scholomance_loot_%28original%29
Also as a side note, i have noticed that this vanilla loot table consists through most if not all the old lv60 dungeons that were downgraded in level.

PS i havent included many wowhead links in this report as 99% of them will be simply useless in this case and cause problems like that occured with Jandice Barov (2).

**Comment by xolon** [comment]
door to ras opens when you kill all other bosses in the instance

**Comment by Helbringer** [comment]
Ok thanks, i will check that.

**Comment by Helbringer** [comment]
Yep your correct, i killed rattlegore and i think jandice barov as well and the door opened. Thats one less bug however Jandice is worse than i thought.

**Comment by Aival94** [comment]
About [Kirtonos the Herald](http://www.wowhead.com/npc=10506/kirtonos-the-herald) maybe i did not undestand well or you make a mistake.There should not be any invisible wall that stop you from falling out of the instance if knock you off and last time i did it there was no such wall(check wowpedia strategy and wowhead comments)

**Comment by Helbringer** [comment]
Well i say that as i remember previously leveling up my main character and running scholomance, i destinctly remember being impressed that i didnt fall out of the instance when getting knocked back. I doubt that blizzard would just let people fly out of the instance, idk maybe im wrong. I will try to find out for sure.

**Comment by 1Andas** [comment]
Another thing to add here, the lever at Jandice Barov's room that is supposed to open a door to the Old Treasure Chest is broken.
